upgrade/rendering-changes/4-0-0.md

```diff
@@ -0,0 +1,233 @@
+---
+title: 4.0.0
+description: Handle the rendering changes in the 4.0.0 release of the Telerik UI for Blazor components.
+page_title: Rendering Changes in 4.0.0
+slug: rendering-changes-in-4-0-0
+position: 3
+---
+
+# Rendering Changes in 4.0.0
+
+* [Calendar](#calendar)
+* [Card](#card)
+* [ChunkProgressBar](#chunkprogressbar)
+* [Common - Date pickers](#common---date-pickers)
+* [Common - Selects](#common-selects)
+* [Dialog](#dialog)
+* [Editor](#editor)
+* [FileSelect](#fileselect)
+* [Form](#form)
+* [Gantt](#gantt)
+* [Grid](#grid)
+* [ListView](#listview)
+* [MultiSelect](#multiselect)
+* [Notification](#notification)
+* [Pager](#pager)
+* [ProgressBar](#progressbar)
+* [Splitter](#splitter)
+* [Toolbar](#toolbar)
+* [TreeList](#treelist)
+* [Upload](#upload)
+* [Window](#upload)
+
+
+## Calendar
+
+* Changed `<span class="k-calendar-nav-today k-today">` to a flat button (`<TelerikButton FillMode="@ThemeConstants.Button.FillMode.Flat">`).
```

Comment:
```suggestion
* Changed `<span class="k-calendar-nav-today k-today">` to a flat button (`<button class="k-button k-calendar-nav-today k-button-flat k-rounded-md k-button-md k-button-flat-primary"><span class="k-button-text">Today</span></button>`).
```

let's use the actual HTML markup that results from the change, rather than the razor markup, which is internal anyway.